src/views/parents/parents.jsx

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var FormattedMessage = require('react-intl').FormattedMessage;
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var Box = require('../../components/box/box.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+
+require('./parents.scss');
+
+var Parents = React.createClass({
+    type: 'Parents',
+    render: function () {
+        return (
+            <div className="inner parents">
+                <h1><FormattedMessage id='general.forParents' /></h1>
+
+                <div className="masthead">
+                    <div>
+                        <p><FormattedMessage id='parents.introOne' /></p>
+                        <p><FormattedMessage id='parents.introTwo' /></p>
+
+                        <ul>
+                            <li><a href = "/about/"><FormattedMessage id="parents.introAbout" /></a></li>
+                            <li><a href = "/educators/"><FormattedMessage id="parents.introEducators" /></a></li>
+                        </ul>
+                    </div>
+
+                    <div>
+                        <iframe
+                            title="Mitch TED Talk"
+                            src="//embed-ssl.ted.com/talks/mitch_resnick_let_s_teach_kids_to_code.html"
+                            frameBorder="0"
+                            webkitAllowFullScreen
+                            mozallowfullscreen
+                            allowFullScreen />
+                    </div>
+                </div>
+                
+                <div className="bodyMain">
```

Comment:
Rather than wrapping each `FlexRow` in a `div` that's an inline block, you can actually use `FlexRow` elements to your advantage, as they'll handle stacking horizontally for you. This could also potentially help to reduce some of the styling you added below by offloading it to the `FlexRow` component to handle it for you, i.e. you'd probably no longer need the `inline-block` formatting.

For instance, on this year's conference sub-site, on the [`expect`](https://scratch.mit.edu/conference/expect) page, we use flex row to horizontally stack two elements of uneven widths (see [here](https://github.com/LLK/scratch-www/blob/develop/src/views/conference/expect/expect.jsx#L23)).

I'd suggest using those uneven styles, and maybe having both the questions and the side elements be within a `FlexRow` and using those ["uneven" stylings](https://github.com/LLK/scratch-www/blob/develop/src/components/flex-row/flex-row.scss#L15-L39).